_common.scss

```diff
@@ -294,4 +295,101 @@
             margin-top: 0.25rem;
         }
     }
+
+    $ltrOutlinePadding: 0 12px 0 16px;
```

Comment:
This file is getting a bit out of hand, it might make sense to split this up similar to the mixin folder.